config.yml

```diff
@@ -204,31 +204,31 @@ php:
       APCU_VERSION: 5.1.17
       CI_HELPER_VERSION: *CI_HELPER_VERSION
       COMPOSER_VERSION: 1.9.0
-      MEMCACHED_VERSION: 3.1.3
+      MEMCACHED_VERSION: 3.1.4
       MODD_VERSION: *MODD_VERSION
       PHP_CS_FIXER_VERSION: 2.15.3
       REDIS_VERSION: 5.0.2
       SECURITY_CHECKER_VERSION: 6.0.2
-      SSH2_VERSION: 1.1.2
+      SSH2_VERSION: 1.2
```

Comment:
Nice, could you please try to uncomment the installation of ssh2 in Dockerfile (and so re-add it in `test.php`)?

Reply:
> Could not download from "https://pecl.php.net/get/ssh2-1.2.tgz", cannot download "pecl/ssh2"

The link is on the download page, but it's dead... https://pecl.php.net/package/ssh2

Reply:
Ok thanks, so let's wait